packages/bpk-component-input/src/bpk-input.scss

```diff
@@ -33,6 +33,25 @@
     @include bpk-input--large;
   }
 
+  &--clearable {
+    @include bpk-input--clearable;
+
+    &__container {
+      @include bpk-input--clearable__container;
+    }
+
+    &__clear-button {
+      @include bpk-input--clearable__clear-button;
+      opacity: 0;
```

Comment:
I left the `opacity` part out of the mixin because it's useless without the sibling selector rule below and I don't think that's something that can be done automatically with Sass?